lib/chef_handler_foreman/foreman_facts.rb

```diff
@@ -12,15 +12,22 @@
 #along with this program.  If not, see <http://www.gnu.org/licenses/>
 
 require 'chef/handler'
+require 'digest/md5'
 
 module ChefHandlerForeman
   class ForemanFacts < Chef::Handler
     attr_accessor :uploader
+    attr_writer   :blacklist
+    attr_writer   :whitelist
```

Comment:
ideally make these three `attr_accessor` instead of `attr_writer`, other classes might find reading it useful